pkgs/desktops/deepin/go-package/dde-daemon/0003-search-in-XDG-directories.patch

```diff
@@ -140,30 +161,6 @@ index 8863d6c2..dd6f8e16 100644
  }
  
  // get user home
-diff --git a/dock/desktop_file_path.go b/dock/desktop_file_path.go
-index 7adc9f55..e1a97679 100644
---- a/dock/desktop_file_path.go
-+++ b/dock/desktop_file_path.go
-@@ -7,6 +7,8 @@ package dock
- import (
- 	"path/filepath"
- 	"strings"
-+
-+	"github.com/adrg/xdg"
- )
- 
- var pathDirCodeMap map[string]string
-@@ -20,6 +22,10 @@ func initPathDirCodeMap() {
- 		"/usr/local/share/applications/": "/L@",
- 	}
- 
-+	for _, dataPath := range xdg.DataDirs {
-+		pathDirCodeMap[dataPath] = "/S@"
```

Comment:
This patch don't work,  although `/run/current-system/sw/share` should not be used directly, let's temporarily use


This function has been rewritten in c++: https://github.com/linuxdeepin/dde-appearance